### DISCUSSION TITLE
iobroker mit Jarvis über Cloud

### DISCUSSION THREAD

**Discussion by orhan77971** [Zefau/ioBroker.jarvis#2489] [Q&A]
Hallo an Alle,

Im Rahmen des Weihnachtsangebotes habe ich den Cloud Zugriff von Iobroker für 1 Jahr gekauft.
Nun möchte ich gerne auf meine Jarvis-Oberfläche zugreifen. Leider klappt das bisher nicht. Einen Remote-Zugriff habe ich zwar bereits über VNC-Viewer, aber das ist äußerst langsam... Ich bekomme Internet über Cable und daher ist ein Zugriff ins LAN über die Fritzbox ebenfalls nicht möglich. Daher wollte ich nun diese Variante probieren.

Hat es schonmal jemand geschafft und kann mir erklären wie man Jarvis über die Cloud aufrufen kann?
Die bisherigen Threads habe ich gelesen aber ich konnte sie entweder nicht verstehen oder führten ins Leere.

Vielen Dank für eure Mühe

**Comment by TZanke** [comment]
Punkt 3 dieser Anleitung müsste für dich doch passen:
https://www.verdrahtet.info/2022/10/15/iobroker-von-unterwegs-steuern-so-klappt-es/
(selbst nicht probiert)
